recipes/coda/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "2.22.1" %}
+{% set sha256 = "dd1cd0112d09cea29afa1568d644f54eccffdad2f5909708b519c1d7c647e9f1" %}
+{% set tag = version %}
```

Comment:
Not needed 
```suggestion
```

Reply:
thanks for the feedback! ^^ making changes..